### PR TITLE
Update Answers.md file

### PR DESCRIPTION
**Pull request by RedHoodJT1988**

### CODE REVIEW

**Comment by RedHoodJT1988** [comment]
I am a little bit confused as to why my media query isn't working. Can you take a look and offer any guidance?

File: css/index.css [hunk 1]
```diff
@@ -137,10 +139,40 @@ footer nav {
...
-    font-size: 14px;
+    font-size: 0.875rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+@media (max-width: 1000px) {
```
Comment (StevieIsmagic) [inline css/index.css]: I notice you added another class to specify even boxes(i.e.,  .box2 - .box10) . Did you add these classes in your html too?

File: css/index.css [hunk 2]
```diff
@@ -137,10 +139,40 @@ footer nav {
...
+
+    .middle-content .boxes .box .box6 {
+        background-color: hotpink;
+    }
+
+    .middle-content .boxes .box .box8 {
+        background-color: indigo;
+    }
+
+    .middle-content .boxes .box .box10 {
```
Comment (StevieIsmagic) [inline css/index.css]: Also try removing the .box class from each of these, so for example you'll only have: ``` .middle-content .boxes box2 {
          background-color: forestgreen;
}```

**Comment by RedHoodJT1988** [comment]
I did. I realized they were missing. And was able to figure out how to get
the backgrounds to work as well. I will be commuting my changes here in a
minute.

File: index.html [hunk 3]
```diff
@@ -14,8 +14,16 @@
 
 <body>
     <header>
-        <nav></nav>
+        <img src="MacintoshHD/Users/jonathanreeves/lamdbaschools/Sprint-Challenge--UI-Responsive/img/lambda-black.png">
```
Comment (StevieIsmagic) [inline index.html]: You were very close here. Try to specify just the relative path of the css file from your html file > ``` ../img/lambda-black.png ```

Comment (RedHoodJT1988) [reply index.html]: That helped. Thank you. I couldn't figure out why it wasn't working.

**Pull request closed without merging**